biz.aQute.bnd.maven/src/aQute/bnd/maven/lib/resolve/BndrunContainer.java

```diff
@@ -176,7 +176,9 @@ public FileSetRepository getFileSetRepository() throws Exception {
 
 		DependencyResolver dependencyResolver = new DependencyResolver(project, repositorySession, resolver, system, scopes);
 
-		return fileSetRepository = dependencyResolver.getFileSetRepository(project.getName(), bundles, useMavenDependencies);
+		String name = (project.getName() != "") ? project.getName() : project.getArtifactId();
```

Comment:
I think this is better as:

```java
String name = project.getName().isEmpty() ? project.getArtifactId() : project.getName();
```

Reply:
sure

Reply:
`!= ""` is a pretty specific test since not all empty strings are the interned empty string.

Reply:
got it, already updated.

Reply:
note that I also pushed a formatting only commit in here to sync up settings with other workspace projects, so be advised. I could pull it out if it confuses the reset.